schema/install_schema.go

```diff
@@ -18,7 +19,37 @@ type InstallSchema struct {
 	GrubOptionsSchema   `json:"grub_options,omitempty"`
 	Image               string `json:"image,omitempty" description:"Use a different container image for the installation"`
 	PowerManagement
-	SkipEncryptCopyPlugins bool `json:"skip_copy_kcrypt_plugin,omitempty"`
+	SkipEncryptCopyPlugins bool                `json:"skip_copy_kcrypt_plugin,omitempty"`
+	Partitions             ElementalPartitions `json:"partitions,omitempty" mapstructure:"partitions"`
+	Firmware               string              `json:"firmware,omitempty" mapstructure:"firmware"`
+	PartTable              string              `json:"part-table,omitempty" mapstructure:"part-table"`
+	GrubDefEntry           string              `json:"grub-entry-name,omitempty" mapstructure:"grub-entry-name"`
+	ExtraPartitions        []*Partition        `json:"extra-partitions,omitempty" mapstructure:"extra-partitions"`
+	Force                  bool                `json:"force,omitempty" mapstructure:"force"`
+	ExtraDirsRootfs        []string            `json:"extra-dirs-rootfs,omitempty" mapstructure:"extra-dirs-rootfs"`
+	Active                 v1.Image            `json:"system,omitempty" mapstructure:"system"`
+	Recovery               v1.Image            `json:"recovery-system,omitempty" mapstructure:"recovery-system"`
+	Passive                v1.Image            `json:"passive,omitempty" mapstructure:"recovery-system"`
+}
+
+type Partition struct {
```

Comment:
only a subset of this is supported to be overridable, if this is the source of truth then we need to make sure to drop fields that are not used. i.e. we dont validate the label or flags